packages/redux-resource/src/utils/set-resource-meta.js

```diff
@@ -26,7 +26,15 @@ export default function setResourceMeta(options) {
     // the user with meta **and** attribute update problems. If the ID check
     // is moved into the success reducers directly, then we may be able to
     // remove these typeof checks for efficiency.
-    if (!id || (typeof id !== 'string' && typeof id !== 'number')) {
```

Comment:
Hey!  Glad someone is on this.  This patch should work, since I had locally patched it this way:

```
if ((!id && id !== 0) || (typeof id !== 'string' && typeof id !== 'number')) {
```

So, if falsy but not because of strict equality with zero.

Reply:
Thanks @ianmstew ! I like the terseness of your snippet. I swapped that in :v: